### ISSUE TITLE
zone.js latest version

### ISSUE THREAD

**Issue by eun-choi** [angular/angular#39094]
🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑

I know that 0.11.2 version of zone.js has been released through the following commit : [a6971ba89adc253bfa4260036ee4a1e0bd76159f](https://github.com/angular/angular/commit/a6971ba89adc253bfa4260036ee4a1e0bd76159f)

However, There is no version 0.11.2 on NPM site yet.
https://www.npmjs.com/package/zone.js

Is it intended?

🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑

**Comment by JiaLiPassion** [comment]
The new release will be available very soon, please wait for a little while.

**Comment by crfrolik** [comment]
Is it possible to change the `@angular/core` peer dependency of zone.js so there is no warning when using zone.js 0.11.X?

**Comment by hfournier** [comment]
I just tried to update from 10.1.1 to 10.1.5 using:
```
ng update @angular/core
```
and got:
```
                  Package "@angular/core" has an incompatible peer dependency to "zone.js" (requires "~0.10.3", would install "0.11.1")
× Migration failed: Incompatible peer dependencies found.
```
so it's not just a warning now

**Comment by bboydflo** [comment]
I've got the same issue as @hfournier

**Comment by JiaLiPassion** [comment]
zone.js `0.11.2` has been released, and about the `peer denedency warning`, it will be updated soon.

**Comment by muuvmuuv** [comment]
Is it save to always use latest zone.js with latest Angular?

**Comment by JiaLiPassion** [comment]
@muuvmuuv , it is safe to use the latest version, if there is breaking changes that impact the existing apps, it will follow the same policy with other angular packages.

**Comment by AndrewKushnir** [comment]
Closing this ticket based on the comments above (zone.js version that should resolve the problem is released). Please reopen if the problem still exists.

**Comment by hfournier** [comment]
I can confirm that I was able to update to the latest @angular/core 10.2.1, but when try:
```
ng update zone.js
```
I get:
```
Fetching dependency metadata from registry...
                  Package "@angular/core" has an incompatible peer dependency to "zone.js" (requires "~0.10.3", would install "0.11.2").
× Migration failed: Incompatible peer dependencies found.
Peer dependency warnings when installing dependencies means that those dependencies might not work correctly together.
You can use the '--force' option to ignore incompatible peer dependencies and instead address these warnings later.
```

**Comment by AndrewKushnir** [comment]
Hi @hfournier, thanks for the feedback.
It looks like the problem still exists, so I'm re-opening this issue.

**Comment by napei** [comment]
This still seems to happen as the peer dependency of @angular/core is not set to accept the latest version of zone

`Package "@angular/core" has an incompatible peer dependency to "zone.js" (requires "~0.10.3", would install "0.11.3")`

**Comment by Splaktar** [comment]
I just raised this with the team to try and get some clarification and hopefully a resolution here.

**Comment by Splaktar** [comment]
Fixed in https://github.com/angular/angular/commit/c43267b91244ac5a3d9f8cc20d909888f5aa0d99 which should be released in `@angular/core@11.1.0`.

**Comment by kbradl16** [comment]
@Splaktar doesnt look like this was ever published?

**Comment by Splaktar** [comment]
@kbradl16 not yet, there should be an `11.1.0-rc.0` release sometime next week.

**Comment by Splaktar** [comment]
@kbradl16 this is available in `11.1.0-next.1` and today's `11.1.0-next.4` preview release.

**Comment by JoostK** [comment]
The peer dependency was updated in #39809, so closing as resolved.
